client/components/status/status.js

```diff
@@ -136,6 +136,8 @@ export default class Status extends React.Component {
             this.loginFormRef.current.submit();
             setUserData({...userData, justAuthenticated: false});
           }
+          // waiting for 1 second after submitting captive portal login form
+          await new Promise((res) => setTimeout(res, 1000));
```

Comment:
we do not know whether it will take 1 second, less or more.
We have to find a better way to do this. I think we have to add an onload eventl listener to the iframe.
I think we have to do this both for login and logout, with a timeout which defaults to 5 seconds and if reached gives a specific error message, something like (captive portal is not reachable).

Reply:
Ok, I will add this change.